app/pages/sections/order-items/additional-services/select/recipient/contextCreator.js

```diff
@@ -2,10 +2,17 @@ import manifest from './manifest.json';
 import { baseUrl } from '../../../../../../config';
 import { getSectionErrorContext } from '../../../../getSectionErrorContext';
 
-// eslint-disable-next-line max-len
-export const backLinkHref = (additionalServicePrices, orderId) => (((additionalServicePrices || {}).prices || {}).length === 1
-  ? `${baseUrl}/organisation/${orderId}/additional-services/select/additional-service`
-  : `${baseUrl}/organisation/${orderId}/additional-services/select/additional-service/price`);
+export const backLinkHref = (req, additionalServicePrices, orderId) => {
+  const { referer } = req.headers;
```

Comment:
I'm wondering if it might be cleaner to do `const { referer } = req.headers || {};` here so we only need to pass `{}` on line 39.

What do you think?

Reply:
Good point - done.